contracts/accountants/HealthCheckAccountant.vy

```diff
@@ -0,0 +1,502 @@
+        work across all vaults of the same underlying.
+    @param vault The vault to redeem from.
+    @param amount The amount in the underlying to withdraw.
+    """
+    assert msg.sender == self.fee_manager, "not fee manager"
+    IVault(vault).withdraw(amount, self, self)
+
+
+@external
+def distribute(token: address) -> uint256:
+    """
+    @notice used to withdraw accumulated fees to the designated recipient.
+    @dev This can be used to withdraw the vault tokens or underlying tokens
+        that had previously been withdrawn.
+    @param token The token to distribute.
+    @return The amount of token distributed.
+    """
+    assert msg.sender == self.fee_manager, "not fee manager"
+
+    rewards: uint256 = ERC20(token).balanceOf(self)
+    self._erc20_safe_transfer(token, self.fee_recipient, rewards)
+
+    log DistributeRewards(token, rewards)
+    return rewards
+
+
+@internal
+def _erc20_safe_transfer(token: address, receiver: address, amount: uint256):
+    # Used only to send tokens that are not the type managed by this Vault.
+    assert ERC20(token).transfer(receiver, amount, default_return_value=True), "transfer failed"
+
+
+@external
+def set_future_fee_manager(future_fee_manager: address):
+    """
+    @notice Step 1 of 2 to set a new fee_manager.
+    @dev The address is set to future_fee_manager and will need to
+        call accept_fee_manager in order to update the actual fee_manager.
+    @param future_fee_manager Address to set to future_fee_manager.
+    """
+    assert msg.sender == self.fee_manager, "not fee manager"
+    assert future_fee_manager != empty(address), "ZERO ADDRESS"
+    self.future_fee_manager = future_fee_manager
+
+    log SetFutureFeeManager(future_fee_manager)
+
+
+@external
+def accept_fee_manager():
+    """
+    @notice to be called by the future_fee_manager to accept the role change.
+    """
+    assert msg.sender == self.future_fee_manager, "not future fee manager"
+    self.fee_manager = self.future_fee_manager
+    self.future_fee_manager = empty(address)
+
+    log NewFeeManager(msg.sender)
+
+
+@external
+def set_fee_recipient(new_fee_recipient: address):
+    """
+    @notice Set a new address to receive distributed rewards.
+    @param new_fee_recipient Address to receive distributed fees.
+    """
+    assert msg.sender == self.fee_manager, "not fee manager"
+    assert new_fee_recipient != empty(address), "ZERO ADDRESS"
+    old_fee_recipient: address = self.fee_recipient
+    self.fee_recipient = new_fee_recipient
+
+    log UpdateFeeRecipient(old_fee_recipient, new_fee_recipient)
+
+
+@view
+@external
+def performance_fee_threshold() -> uint16:
+    """
+    @notice External function to get the max a performance fee can be.
+    @return Max performance fee the accountant can charge.
+    """
+    return self._performance_fee_threshold()
+
+
+@view
+@internal
+def _performance_fee_threshold() -> uint16:
```

Comment:
These could be public constants `PERFORMANCE_FEE_THRESHOLD` and `MANAGEMENT_FEE_THRESHOLD`. 
may be simpler compared to the current functions with hardcoded return values. what do you think?